about.html

```diff
@@ -11,12 +11,11 @@
     <header>
         <nav class="head">
             <i class="fa fa-bars" id="open_menu" aria-hidden="true"></i>
-         <a href="home.html"><img src="./Features/images/project-logo.png" alt="logo" class="logo"></a>
         </nav>
         <ul class="mobile-nav">
             <i class="fa fa-times" id="close_menu" aria-hidden="true"></i>
             <li class="menu-items"><a href="about.html">About</a></li>
-            <li class="menu-items"><a href="#">Program</a></li>
+            <li class="menu-items"><a href="index.html">Program</a></li>
```

Comment:
- [optional] kindly ensure that the mobile menu disappears when switched to the desktop version. Also, consider adding a link to the home page in your mobile menu when in the about page. 
![Screenshot (1178)](https://github.com/Bupilipili/First-Capstone-Project/assets/65298048/5de56733-56f0-4273-b7fb-c294838fd2f9)